kafka-compose.yaml

```diff
@@ -53,3 +53,12 @@ services:
       KAFKA_BROKERCONNECT: "kafka1:19092"
     depends_on:
       - "kafka1"
+
+  redis:
```

Comment:
Why not use the (opinionated, but its opinions are, I think correct for us) redis chart in lsst-sqre/charts ?

Reply:
Ah, OK, because this is for the test suite, not the _actual_ orchestration.

Reply:
Right, Phalanx deployment/charts are at https://github.com/lsst-sqre/phalanx/pull/3014 and we're using the internal redis chart there.